stripe/src/main/java/com/stripe/android/googlepay/StripeGooglePayContract.kt

```diff
@@ -128,3 +129,31 @@ internal class StripeGooglePayContract :
         }
     }
 }
+
+fun Status.getErrorMessage(): String? {
+    return when (this) {
+        Status.RESULT_SUCCESS -> null
+        Status.RESULT_INTERNAL_ERROR, Status.RESULT_CANCELED, Status.RESULT_DEAD_CLIENT -> "An internal error occurred."
+        else -> {
+            when (this.statusCode) {
+                CommonStatusCodes.API_NOT_CONNECTED, // "The client attempted to call a method from an API that failed to connect."
+                CommonStatusCodes.CANCELED, // -> "The result was canceled either due to client disconnect or PendingResult.cancel()."
+                CommonStatusCodes.DEVELOPER_ERROR, // -> "The application is misconfigured."
+                CommonStatusCodes.ERROR, // -> "The operation failed with no more detailed information."
+                CommonStatusCodes.INTERRUPTED, // -> "A blocking call was interrupted while waiting and did not run to completion."
+                CommonStatusCodes.INVALID_ACCOUNT, // -> "The client attempted to connect to the service with an invalid account name specified."
+                CommonStatusCodes.SERVICE_DISABLED, // -> "This constant is deprecated. This case handled during connection, not during API requests. No results should be returned with this status code."
+                CommonStatusCodes.SERVICE_VERSION_UPDATE_REQUIRED, // -> " This constant is deprecated.This case handled during connection, not during API requests . No results should be returned with this status code."
+                CommonStatusCodes.SUCCESS, // -> "The operation was successful.SUCCESS_CACHE The operation was successful, but was used the device's cache."
+                CommonStatusCodes.INTERNAL_ERROR -> "An internal error occurred."
+                CommonStatusCodes.RESOLUTION_REQUIRED -> "Completing the operation requires some form of resolution."
```

Comment:
I haven't been able to reproduce when this can happen, but it is possible:  See the handleError function here:
https://developers.google.com/pay/api/android/guides/tutorial#example